PUSH_NOTIFICATION_DATA_GUIDE.md

```
@@ -0,0 +1,129 @@
+      "title": "Sample title for a Klaviyo push notification",
+      "body": "Sample body for a Klaviyo push notification"
+    },
+    "mutable-content": 1
+  },
+  "rich-media": "https://picsum.photos/200/300.jpg",
+  "rich-media-type": "jpg"
+}
+```
+
+**Video:**
+```json
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)